presto-parser/src/main/java/com/facebook/presto/sql/tree/QualifiedName.java

```diff
@@ -81,12 +81,13 @@ public String toString()
      */
     public Optional<QualifiedName> getPrefix()
     {
-        if (parts.size() == 1) {
+        if (originalParts.size() == 1) {
```

Comment:
Should we consider getting rid of parts/originalParts altogether in this PR and just accept the parts as-is from the caller? We lowercase in the constructor of this class. I feel like we already moved to prevent lowercasing elsewhere

Reply:
So QualifiedName, maintains both parts -> which convert the supplied value to lower case & originalParts which keeps the original value name of same QualifiedName. I think it should be Ok, to preserve both as of now, this PR we have used originalParts from QualifiedName in the required place to keep the case original for Schema & Table names.